modules/ROOT/pages/upgrade-message-flows.adoc

```diff
@@ -1,8 +1,8 @@
 = Template Version Upgrades
 
-Partner Manager leverages prebuilt templates that convert your B2B message flow configurations into runtime applications. These templates are periodically updated to add new features and enhancements. You can upgrade your message flows to use the updated templates to take advantage of the new capabilities that enable you to better integrate with your trading partner ecosystem.
+Partner Manager leverages prebuilt templates that convert your B2B message flow configurations into runtime applications. These templates are periodically updated to add new features and enhancements. You can upgrade your message flows to use the updated templates and to take advantage of the new capabilities that enable better integration with your trading partner's ecosystem.
```

Comment:
"leverages" always makes me wonder "from what source," so I avoid it if a source is not named.
```suggestion
Partner Manager uses prebuilt templates to convert your B2B message flow configurations into runtime applications. As these templates are updated, you can upgrade your message flows to use them to take advantage of new capabilities that enable better integration with your trading partner's ecosystem.
```
The text in this document uses SO MANY different versions of naming for features. As you become more familiar with the product, I hope you can figure out how to distill them to one word or phrase apiece. For example, "your B2B message flow configurations": do we use B2B, or do we belabor you + partner or your organization + your partner? Are these messages or flows or transmissions or configurations? Makes my head spin. ;-)